docs/gem5-sphinx-apidoc

```diff
@@ -0,0 +1,16 @@
+import re
+import sys
+
+extensions = ['myst_parser']
+
+source_suffix = {
+    # '.rst': 'restructuredtext',
+    # '.txt': 'restructuredtext',
```

Comment:
Why are these commented out?

Reply:
Those I just removed, replacing those lines with markdown instead fixed a lot of formatting issues.